README.md

```diff
@@ -45,6 +45,14 @@ You can manually specify any of the options on the fly:
 
 E.g.: `gad -o Elao -r symfony-standard -u Tom32i -p MyGitHubPersonalAccessToken`
 
+### On Github
```

Comment:
I'd suggest `Configure your agile project on GitHub`.
And I'd move this section just below the `## Usage` title, because it's the first step you need to take :)